packages/story-editor/src/components/autoSaveHandler/index.js

```diff
@@ -58,7 +54,7 @@ function AutoSaveHandler() {
     );
 
     return () => clearTimeout(timeout);
-  }, [autoSaveInterval, hasNewChanges, isUploading, story, pages]);
+  }, [autoSaveInterval, hasNewChanges, isUploading, isAutoSaving]);
```

Comment:
Why do we need isUploading here?

Reply:
Because we prevent saving while any upload is in progress. Always has been the case. Avoids saving data with half-finished uploads.

Reply:
So to confirm, if a user uploads a video and it is transcoding, there will be no autosaves? If you upload say 5 videos, that could take a while. Maybe we could add a check to see if there an process media element on them. If not do an autosave.

Reply:
Yes, but IIRC only if the video is uploaded to the canvas and not just the library.

Reply:
Just asking, as if you transcoding video takes say, 5 minutes, then blocking autosaves for that time, that could be a serious problem.